ncl/ncl_entries/dewtemp.ipynb

```diff
@@ -0,0 +1,113 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# dewtemp_trh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "NCL's [`dewtemp_trh`](https://www.ncl.ucar.edu/Document/Functions/Built-in/dewtemp_trh.shtml) calculates the dew point temperature given temperature and relative humidity using the equations from John Dutton's _\"Ceaseless Wind\"_ (pg. 273-274) and returns a temperature in Kelvin\n",
+    "\n",
+    "<div class=\"admonition alert alert-info\">\n",
```

Comment:
This seems to work fine, but you should also be able to do these via the MystMD approach, which is a bit more concise: https://mystmd.org/guide/admonitions

Reply:
I had previous tried to do this with 
```
:::{important}
To convert from Kelvin to Celsius `-273.15` and to convert from Celsius to Kelvin `+273.15`
:::
```
But it wasn't recognized by the Jupyter markdown, so it just displayed as:
`:::{important}
To convert from Kelvin to Celsius '-273.15' and to convert from Celsius to Kelvin '+273.15'
:::`

Reply:
I'm currently using the `environment.yml` with an additional `conda install jupyterlab`. Does this also need `jupyter-myst`?